_lifeofinternatpuppet/post_15.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: Week 15
+author: Disha-maker
+categories:
+  - team
+  - basics
+tags:
+  - week 15
+---
+
+Hello everyone,
+
+What a busy time of the year as we wrap up for Christmas?
+
+Important Highlights from this week:-
+- I continued working on running Litmus Acceptance Tests locally for the IAC modules before shifting them on the new Google Cloud CI.
+The modules tested this week are given below:-
+    - [puppetlabs-vcsrepo](https://github.com/puppetlabs/puppetlabs-vcsrepo)
+    - [puppetlabs-exec](https://github.com/puppetlabs/puppetlabs-exec)
+    - [puppetlabs-facter_task](https://github.com/puppetlabs/puppetlabs-facter_task)
+
+- Thank you to [Sheena](https://github.com/sheenaajay) for doing such an enjoyable session on module release prep and module release.
+The modules covered during the session were:-
+    - [puppetlabs-registry](https://github.com/puppetlabs/puppetlabs-registry) - (3.2.0)
+    - [puppetlabs-apt](https://github.com/puppetlabs/puppetlabs-apt) - (7.7.0)
+
+- Thanks to [Daiana](https://github.com/daianamezdrea) and [Paula](https://github.com/pmcmaw)for giving me the following PR's for review and merge
+    - PDK update for GA workflows[https://github.com/puppetlabs/puppetlabs-helm/pull/134]
+    - PDK update for GA workflow [https://github.com/puppetlabs/puppetlabs-kubernetes/pull/457]
+    - Delete i18n header(https://github.com/puppetlabs/iac/pull/153)
```

Comment:
These do not look like properly formatted links. What is the intention for these? Is it to look like this?

[PDK update for GA workflows](https://github.com/puppetlabs/puppetlabs-helm/pull/134)

Again, please note [my comment here](https://github.com/puppetlabs/iac/pull/155#pullrequestreview-550000540). Building and testing this locally would catch any formatting errors like this.

Reply:
@sanfrancrisko, I just added them for reference, but if you think that they are not necessary, then I will remove them. Personally, I don't think they were needed but just included them anyway :) Let me know either way of what to do!

Reply:
I am just giving names of some of the PR's that I have reviewed and merged, but can include them as normal text and not hyperlinks with the https address removed too :)

Reply:
You can include them if you want, but I'm not sure why you would enclose them in square braces, which made me think you intended these to be a hyperlink in MD format, e.g.
```ruby
[PDK update for GA workflow](https://github.com/puppetlabs/puppetlabs-kubernetes/pull/457)
```